parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala

```diff
@@ -69,7 +51,9 @@ object Tokens extends CrfTokenable {
   private def normalizeInput(input: String): String = {
     val upperInput = input.toUpperCase()
 
-    val inputWithoutCounties = removeCounties(upperInput)
+    val inputWithoutAccents = StringUtils.stripAccents(upperInput)
```

Comment:
Looking a the StringUtils javadoc https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html I see it has methods for Levenshtein, Jaro-Winkler, Fuzzy and other string comparison techniques which might be useful for Hopper Score calculations.